src/core/public/chrome/ui/header/header.scss

```diff
@@ -18,7 +18,6 @@
 
   &.primaryApplicationHeader {
     padding-top: $euiSizeM - $euiSizeXS;
-    border-bottom: 1px solid $euiColorLightShade;
```

Comment:
was this just unnecessary or what's the before/after look like for this border? would this impact current headers (before page headers) in any way?

Reply:
I made this change along with adding the `EuiHorizontalRule`,  in order to let the line separator always stays at the bottom, to make it align with UX. This shouldn't affect current legacy headers, since this class is only added for new application header